includes/class-wc-siftscience-admin.php

```diff
@@ -74,9 +74,16 @@ public function output_settings_fields() {
 			}
 		}
 
+		private function get_auto_text_style(){
+			return '<style type="text/css">label[for="'
+				. WC_SiftScience_Options::$send_on_create_enabled . '"]+p'
```

Comment:
If you set this value to a smaller variable name ($id for example) before the return statement, you can then use string interpolation for the style tag and it will be much easier to read and understand.